### PR TITLE
Set git user/email settings in Dockerfile

### PR DESCRIPTION
**Pull request by zachkirsch**

### CODE REVIEW

File: packages/generators/sdk/cli/docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM node:18.1.0-alpine
 
 RUN apk --no-cache add git
+git config --global user.name "fern"
+git config --global user.email "hey@buildwithfern.com"
```
Comment (dsinghvi) [inline packages/generators/sdk/cli/docker/Dockerfile:5]: you'll need to add add `&&`

Comment (dsinghvi) [reply packages/generators/sdk/cli/docker/Dockerfile]: multiple runs were cool too

Comment (dsinghvi) [reply packages/generators/sdk/cli/docker/Dockerfile]: i think && is just a little more efficient in terms of docker layers

Comment (zachkirsch) [reply packages/generators/sdk/cli/docker/Dockerfile]: ah cool

**Pull request merged**